integration/common.sh

```diff
@@ -23,23 +23,46 @@ function runTest() {
 function waitForServerStartup() {
```

Comment:
FYI, scripts without the "function" keyword are more portable. Eg, "foo() {}" is better than "function foo() {}"